catalyst_voices/lib/pages/voting/voting_page.dart

```diff
@@ -1,17 +1,190 @@
+import 'package:catalyst_cardano_serialization/catalyst_cardano_serialization.dart';
+import 'package:catalyst_voices/widgets/cards/pending_proposal_card.dart';
+import 'package:catalyst_voices/widgets/common/tab_bar_stack_view.dart';
+import 'package:catalyst_voices_assets/catalyst_voices_assets.dart';
+import 'package:catalyst_voices_localization/catalyst_voices_localization.dart';
+import 'package:catalyst_voices_models/catalyst_voices_models.dart';
+import 'package:catalyst_voices_shared/catalyst_voices_shared.dart';
 import 'package:flutter/material.dart';
 
+final _proposalDescription = """
+Zanzibar is becoming one of the hotspots for DID's through
+World Mobile and PRISM, but its potential is only barely exploited.
+Zanzibar is becoming one of the hotspots for DID's through World Mobile
+and PRISM, but its potential is only barely exploited.
+"""
+    .replaceAll('\n', ' ');
+
+final _proposals = [
+  PendingProposal(
+    id: 'f14/0',
+    fund: 'F14',
+    category: 'Cardano Use Cases / MVP',
+    title: 'Proposal Title that rocks the world',
+    lastUpdateDate: DateTime.now().minusDays(2),
+    fundsRequested: Coin.fromAda(100000),
+    commentsCount: 0,
+    description: _proposalDescription,
+    completedSegments: 0,
+    totalSegments: 13,
+  ),
+  PendingProposal(
+    id: 'f14/1',
+    fund: 'F14',
+    category: 'Cardano Use Cases / MVP',
+    title: 'Proposal Title that rocks the world',
+    lastUpdateDate: DateTime.now().minusDays(2),
+    fundsRequested: Coin.fromAda(100000),
+    commentsCount: 0,
+    description: _proposalDescription,
+    completedSegments: 7,
+    totalSegments: 13,
+  ),
+  PendingProposal(
+    id: 'f14/2',
+    fund: 'F14',
+    category: 'Cardano Use Cases / MVP',
+    title: 'Proposal Title that rocks the world',
+    lastUpdateDate: DateTime.now().minusDays(2),
+    fundsRequested: Coin.fromAda(100000),
+    commentsCount: 0,
+    description: _proposalDescription,
+    completedSegments: 13,
+    totalSegments: 13,
+  ),
+];
+
+final _favoriteProposals = ValueNotifier<List<PendingProposal>>([]);
+
 class VotingPage extends StatelessWidget {
   const VotingPage({super.key});
 
   @override
   Widget build(BuildContext context) {
-    return Container(
-      color: Colors.cyan,
-      alignment: Alignment.center,
-      child: Text(
-        'Voting',
-        style: Theme.of(context).textTheme.titleLarge,
+    return Padding(
+      padding: const EdgeInsets.symmetric(horizontal: 32),
+      child: ListView(
```

Comment:
you should be able set padding as `ListView` parameter

Reply:
Ah true, first I added a column, wrapped it with padding, then replaced column with a list and padding stayed. Fixed.